src/selectedtests/task_mappings/create_task_mappings.py

```diff
@@ -187,23 +186,24 @@ def transform(self) -> List[Dict]:
         task_mappings = []
         for mapping in self.mappings:
```

Comment:
This is unrelated to your change, but could clean up this code some.

All the loops in here look strange to me. There seems to be a pattern of loop over a dictionary, then get the item from the dictionary. Python will do that for us:
```
for source_file, cur_mapping in self.mappings.items():
```